config/default.yml

```diff
@@ -1172,10 +1172,13 @@ Metrics/MethodLength:
 Metrics/BlockLength:
   CountComments: false  # count full line comments?
   Max: 25
-  Exclude:
-    - 'Rakefile'
-    - '**/*.rake'
-    - 'spec/**/*.rb'
+  ExcludedMethods:
```

Comment:
Sure I can do that,

but I want to point out that excluding files ending in `.rake` or living in the folder `spec` also carries some library specfic assumptions.

Should I revert the config to its initial state (i.e. keep the file exclusions) or leave out both, the file exclusions and the excluded methods?